packages/@aws-cdk/aws-kms/lib/key.ts

```diff
@@ -268,6 +268,13 @@ export interface KeyProps {
    * @default RemovalPolicy.Retain
    */
   readonly removalPolicy?: RemovalPolicy;
+
+  /**
+   * Whether the key usage can be granted by IAM policies
+   *
+   * @default false
+   */
+  readonly enablePolicyControl?: boolean;
```

Comment:
Should this be `enableIamPolicyControl` I didn't like the way that reads with the capital `I` and lower `am`. However, I am not sure Policy is enough here because technically there is a Key Policy.